### PR TITLE
Upgrades Atmos APC power cell roundstart

### PR DESCRIPTION
**Pull request by SchrodingersWolf**
## What Does This PR Do
Let's try this again. Remake of #25894

Massively buffs Atmospherics roundstart by changing their APCs to be of the 'Important' subtype, which has 10000 charge compared to 2500 charge.

## Why It's Good For The Game
~~CRUNCH please I don't want to lose my fur~~
Atmospherics tend to run out of power roundstart while Engineering is still setting up the Engine and other power sources - Given that Atmospherics is quite important to SM and Turbine setup, more roundstart power would allow Engineering more time.

## Testing
Loaded up maps, checked that it works.

## Changelog
:cl:
tweak: Atmospherics APCs on all maps now have more charge roundstart
/:cl:

### CODE REVIEW

**Review by lewcc — APPROVED** [review]
Code looks good

**Review by DGamerL — CHANGES_REQUESTED** [review]
Please deconflict for merge
Also, please use a more descriptive PR name

File: _maps/map_files/stations/boxstation.dmm [hunk 1]
```diff
@@ -40181,9 +40181,7 @@
 	d2 = 2;
 	icon_state = "0-2"
 	},
-/obj/machinery/power/apc/directional/north{
-	shock_proof = 1
-	},
+/obj/machinery/power/apc/important/directional/north,
```
Comment (Sirryan2002) [inline _maps/map_files/stations/boxstation.dmm:40184]: why are you removing the shock_proof on these apcs?

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

File: _maps/map_files/stations/boxstation.dmm (hunk 1 above)
Comment (SchrodingersWolf) [reply _maps/map_files/stations/boxstation.dmm:40184]: Consistency - All other stations do not have shock_proof set on Engineering APCs other than the Engine APCs.
Also
Screenshot (3)

**Review by Sirryan2002 — APPROVED** [review]

**Comment by Sirryan2002** [comment]
@DGamerL

**Review by DGamerL — CHANGES_REQUESTED** [review]
PR has been conflicted for a month now

**Review by DGamerL — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**